themes/shijin4/layouts/partials/fundraising.html

```diff
@@ -0,0 +1,21 @@
+<h2 class="fundraising">Fundraising 2018</h2>
```

Comment:
This file should be in the top level "layouts" directory, as it's not theme-specific.